Makefile

```
@@ -124,24 +141,48 @@ else
+# libsais_gsa_omp can run parallel induced-sorting. libomp is already a
+# link dep of bwa-mem2's alignment paths; no new dep.
+LIBSAIS_DIR    = ext/libsais
+LIBSAIS_OBJS   = $(LIBSAIS_DIR)/src/libsais.o $(LIBSAIS_DIR)/src/libsais64.o
+LIBSAIS_CFLAGS = -O3 -std=c99 -DLIBSAIS_OPENMP -I$(LIBSAIS_DIR)/include
+ifeq ($(UNAME_S),Darwin)
+    # Without libomp the brew prefix evaluates to empty, which would silently
+    # produce -I/include / -L/lib and fail later as `ld: library not found
+    # for -lomp`. Fail fast with an actionable hint, mirroring test/Makefile.
+    LIBOMP_PREFIX := $(shell brew --prefix libomp 2>/dev/null)
+    ifeq ($(strip $(LIBOMP_PREFIX)),)
+        $(error libomp not found: install with `brew install libomp`, or set LIBOMP_PREFIX to its install prefix)
```

checkmake (0.3.2)

[warning] Target "$(error libomp not found" should be declared PHONY.

(phonydeclared)